pages/checkin.js

```diff
@@ -0,0 +1,138 @@
+import Image from "next/image";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { locations } from "../db/data";
+import { localStorage } from "../db/localStorage";
+
+export default function CheckIn({
+  newCheckin,
+  setCheckins,
+  checkinTime,
+  setCheckinTime,
+  checkinData,
+}) {
+  const router = useRouter();
+  function handleCheckIn(event) {
+    event.preventDefault();
+    const time = new Date()
+      .toLocaleTimeString()
+      .split(":")
+      .slice(0, 2)
+      .join(":");
+    setCheckinTime(time);
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const checkinData = {
+      id: data.id,
+      checkin_Time: time,
+      name: locations.find((location) => location.id === data.id).name,
+    };
+    setCheckins([newCheckin]);
+
+    localStorage.push(checkinData);
+    router.push(`/location/${data.id}`);
+  }
+
+  return (
+    <>
+      <ImageWrapper>
+        <Image
+          src="/Logo.png"
+          height={200}
+          width={290}
+          alt="A Picture of a Hanger with Text around"
+          priority="high"
+        />
+      </ImageWrapper>
+      <AllInputWrapper>
+        <StyledForm onSubmit={handleCheckIn}>
+          <NumberInputWrapper>
+            <legend>Bügel-Id/qr-code:</legend>
```

Comment:
A `legend` is only used as heading for a `fieldset`, not for a label / input directly. The text "Bügel-Id/qr-code" is to be placed in your `label` below because it's the text for the input field!

Reply:
Hopefully it makes more sense for you if my us goes to QA because you can see that the text otherwise pops to the left